QoL - Adjust redraw light delay to capture holding an arrow key to move.

Azmoria opened the pull request:
While trying to replicate/figure out what was happening in #1122 I noticed that the debounce for redraw light would run before holding a movement key down triggered the next function call. 

For my setup 500ms captures the next call when holding down the key. I think this resolves some of the issue seen in the above PR as it won't stutter before letting go of the key for scenes with lots of walls or lower end pcs.

I'm going to hold off on this and the other improvements by Josh-Archer until next release since we're so close.

Josh-Archer commented:
This definitely helps. I think we still need to break the long running tasks up some. But I think this is a minimum should do while I continue to refine mine and apply your feedback (thanks for that by the way!)

Azmoria commented:
> This definitely helps. I think we still need to break the long running tasks up some. But I think this is a minimum should do while I continue to refine mine and apply your feedback (thanks for that by the way!)

sounds good 👍 - I updated some of my comments and added one where I think the flashing might be. Thanks for looking at all these improvements.
